samwise.js

```diff
@@ -5,10 +5,10 @@
  * Released under the MIT license.
```

Comment:
Good job on updating the documentation along with the changes. It's very thorough and helpful!